trino-connector/src/test/java/com/datastrato/gravitino/trino/connector/TestCreateGravitinoConnector.java

```diff
@@ -51,4 +53,36 @@ public void testCreateSimpleCatalogNameConnector() throws Exception {
 
     server.close();
   }
+
+  @Test
+  public void testCreateConnectorsWithDisableSimpleCatalog() throws Exception {
+    server = new GravitinoMockServer(false);
+    Session session = testSessionBuilder().setCatalog("gravitino").build();
+    QueryRunner queryRunner = DistributedQueryRunner.builder(session).setNodeCount(1).build();
+
+    GravitinoAdminClient gravitinoClient = server.createGravitinoClient();
+    TestGravitinoPlugin gravitinoPlugin = new TestGravitinoPlugin(gravitinoClient);
+    queryRunner.installPlugin(gravitinoPlugin);
+
+    // test create two connector and set gravitino.simplify-catalog-names = true
+    {
+      // create a gravitino connector named gravitino using metalake test
+      HashMap<String, String> properties = new HashMap<>();
+      properties.put("gravitino.metalake", "test");
+      properties.put("gravitino.uri", "http://127.0.0.1:8090");
+      properties.put("gravitino.simplify-catalog-names", "false");
```

Comment:
false? Should it be `true`?

Reply:
Should be `false`

Reply:
Then you should modify the comment `test create two connector and set gravitino.simplify-catalog-names = true`.